venues/thecvf.com/ECCV/2020/Conference/python/emergency_reviews.py

```diff
@@ -0,0 +1,206 @@
+                emergency_reviewer_group.members.append(profile.id)
+
+    x = client.post_group(emergency_reviewer_group)
+    print('posted updated emergency reviewers group')
+    print ('Posting {0} edges'.format(len(emergency_load_edges)))
+    posted_edges = openreview.tools.post_bulk_edges(client, emergency_load_edges)
+    print ('Posted {0} edges'.format(len(posted_edges)))
+
+    map_submissions = {note.number: note for note in openreview.tools.iterget_notes(client, invitation = 'thecvf.com/ECCV/2020/Conference/-/Blind_Submission')}
+    all_reviews = openreview.tools.iterget_notes(client, invitation='thecvf.com/ECCV/2020/Conference/Paper[0-9]*/-/Official_Review')
+    map_paper_to_reviews = {}
+    for review in all_reviews:
+        paper_num = int(review.invitation.split('Paper')[1].split('/')[0])
+        if paper_num in map_submissions:
+            if paper_num not in map_paper_to_reviews:
+                map_paper_to_reviews[paper_num] = []
+            map_paper_to_reviews[paper_num].append(review)
+
+    # Add entries for papers without any reviews
+    for paper_num in map_submissions:
+        if paper_num not in map_paper_to_reviews:
+            map_paper_to_reviews[paper_num] = []
+
+    emergency_demand_invitation = client.post_invitation(openreview.Invitation(
+        id='thecvf.com/ECCV/2020/Conference/Emergency_Reviewers/-/Custom_User_Demands',
+        signatures=['thecvf.com/ECCV/2020/Conference'],
+        readers=[
+            'thecvf.com/ECCV/2020/Conference',
+            'thecvf.com/ECCV/2020/Conference/Area_Chairs'],
+        writers=['thecvf.com/ECCV/2020/Conference'],
+        invitees=['thecvf.com/ECCV/2020/Conference'],
+        reply = {
+            'readers': {'values-copied': [
+                'thecvf.com/ECCV/2020/Conference',
+                'thecvf.com/ECCV/2020/Conference/Area_Chairs',
+                '{tail}']},
+            'nonreaders': {'values-regex': 'thecvf.com/ECCV/2020/Conference/Paper.*/Authors'},
+            'writers': {'values': ['thecvf.com/ECCV/2020/Conference']},
+            'signatures': {'values': ['thecvf.com/ECCV/2020/Conference']},
+            'content': {
+                'head': {
+                    'query' : {'invitation': 'thecvf.com/ECCV/2020/Conference/-/Blind_Submission'},
+                    'type': 'Note'
+                },
+                'tail': {
+                    'query' : {'id': 'thecvf.com/ECCV/2020/Conference/Emergency_Reviewers'},
+                    'type': 'Group'
+                },
+                'weight': {
+                    'value-regex': '[-+]?[0-9]*\\.?[0-9]*'
+                },
+                'label': {
+                    'value-regex': '.*'
+                }
+            }
+        }
+    ))
+    print ('Posted invitation:', emergency_demand_invitation.id)
+
+    emergency_review_demands = []
+    # TODO: Need to add a conflict edge for the reviewers of the existing reviews for a paper ?
```

Comment:
Yes, I think you need to use the current assignments as conflict edges so we don't assign the same reviewer.

Reply:
Made this update. Now we are adding existing paper reviewer assignments as conflicts with the label "Already Assigned". While doing this I noticed that there is an issue as I am not able to delete these specific conflict edges. The `delete_edges` function only takes invitation as input and we definitely do not want to delete all the edges for the conflict invitation. 
Possible solutions:
1. Fix the backend to be able to delete edges by filtering on invitation, head, tail, label
2. I could create a separate script to do post these edges for existing assignments, so that we do not create multiple edges for the same thing when we run the script twice.

Reply:
I have separated the tasks into 2 scripts now. Updated PR description.

Reply:
Ok, another option is to delete the edges, one by one.